cinema_service/settings.py

```diff
@@ -39,13 +39,15 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
```

Comment:
The `debug_toolbar` is included in `INSTALLED_APPS` and `MIDDLEWARE`, which is useful for development but should be removed or disabled in production to prevent performance issues and potential security risks.